scripts/find_orphaned_cables.py

```diff
@@ -0,0 +1,23 @@
+from django.db.models import Q
```

Comment:
This needs to be:

```
from django.db.models import Count, Q
```

otherwsie you get

> NameError: name 'Count' is not defined

Reply:
```suggestion
from django.db.models import Count, Q
```